tests/test_metrics.py

```diff
@@ -89,10 +98,12 @@ def test_average_metric_additions(self):
             self.assertAlmostEqual(actual_output, output, places=6)
             self.assertIsInstance(actual_output, float)
 
-    def test_fixedmetric(self):
```

Comment:
It's not still useful to test FixedMetric here?

Reply:
You're right, I should leave it.